src/data/vault.json

```diff
@@ -145,7 +145,7 @@
       "os": "linux"
     }
   ],
-  "basePaths": ["docs", "api-docs", "intro"],
+  "basePaths": ["docs", "api-docs", "intro", "tutorials"],
```

Comment:
Not sure if we feel like this conflates the intent of this property, but it does allow our current content URL rewrites to "just work" for tutorials links that are authored in the context of dev dot.